css-basics/css-basics_pseudo-park/css/styles.css

```diff
@@ -22,14 +22,31 @@
 
 .animal {
   padding: 16px;
-
+  font-size: 32px;
   /*
     The animals are very small.
     The size of the emojis is controlled via the font size.
     Set the font size to 32px.
   */
 }
 
+.crocodile :hover {
+  background-color: lightblue;
+}
+
+.giraffe {
+  text-align: center;
+}
+.giraffe :hover {
+  background-color: lightblue;
+}
+
+.dino {
+  text-align: right;
+}
+.dino :hover {
```

Comment:
remove the space between the class and the hover eg `.dino:hover` then it should work :-)